includes/wikis.php

```diff
@@ -125,13 +125,13 @@
 		'api' => $baseurl . '/brawlstars/api.php',
 	],
 	'starcraft' => [
-		'name' => 'Brood War',
+		'name' => 'StarCraft: Brood War',
```

Comment:
This one probably should just be StarCraft, we're not even playing Brood War anymore since Remastered dropped.

Reply:
I agree 100%. I just put Brood War there as a compromise solution, thinking that adding something to the current name would be more appreciated than changing the name completely.

Reply:
I'd say let's drop the Brood War part